src/Fixer/Basic/NonPrintableCharacterFixer.php

```diff
@@ -104,7 +104,7 @@ protected function createConfigurationDefinition()
         return new FixerConfigurationResolver([
             (new FixerOptionBuilder('use_escape_sequences_in_strings', 'Whether characters should be replaced with escape sequences in strings.'))
                 ->setAllowedTypes(['bool'])
-                ->setDefault(false) // @TODO 3.0 consider changing to true, but it will require that all fixed code by default is php7+, maybe the best is to keep it in `@PHP70Migration:risky` only
+                ->setDefault(\PHP_VERSION_ID >= 70000)
```

Comment:
```suggestion
                ->setDefault(true)
```

you may explicitly set `true` here.
v3 won't suppor PHP 5.x

Reply:
so at least
```suggestion
                ->setDefault(\PHP_VERSION_ID >= 70000) // @TODO 3.0 replace with `true` when changing PHP requirement to 7+
```